.github/workflows/npm-publish-github-packages.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
```

Comment:
const cloudPaymaster = createPimlicoPaymasterClient({
  chain: baseSepolia,
  transport: http(rpcUrl),
  entryPoint: ENTRYPOINT_ADDRESS_V06,
});

const smartAccountClient = createSmartAccountClient({
  account: simpleAccount,
  chain: baseSepolia,
  bundlerTransport: http(rpcUrl),
  // IMPORTANT: Set up Cloud Paymaster to sponsor your transaction
  middleware: {
    sponsorUserOperation: cloudPaymaster.sponsorUserOperation,
  },
});

Reply:
Thank u